jwst/ami/oifits.py

```diff
@@ -14,42 +14,48 @@
 
 
 class RawOifits:
+    """
+    Store AMI data in the format required to write out to OIFITS files.
+
+    Takes fringefitter class, which contains nrm_list and instrument_data attributes,
+    all info needed to write oifits. Angular quantities of input are in radians from
+    fringe fitting; converted to degrees for saving. Populate the structure needed to
+    write out oifits files according to schema.
+    Produces averaged and multi-integration versions, with sigma-clipped stats over
+    integrations.
+
+    Notes
+    -----
+    Based on ObservablesFromText from ImPlaneIA, e.g.
+    https://github.com/anand0xff/ImPlaneIA/blob/master/nrm_analysis/misctools/implane2oifits.py#L32
+    """
+
     def __init__(self, fringefitter, method="mean"):
         """
-        Class to store AMI data in the format required to write out to OIFITS files.
-
-        Takes fringefitter class, which contains nrm_list and instrument_data attributes,
-        all info needed to write oifits. Angular quantities of input are in radians from
-        fringe fitting; converted to degrees for saving. Populate the structure needed to
-        write out oifits files according to schema.
-        Produces averaged and multi-integration versions, with sigma-clipped stats over
-        integrations.
+        Initialize the RawOifits object.
 
         Parameters
         ----------
-        fringefitter: FringeFitter object
+        fringefitter : FringeFitter object
             Object containing nrm_list attribute (list of nrm objects)
             and other info needed for OIFITS files
-        method: string
+        method : str
             Method to average observables: mean or median. Default mean.
-
-        Notes
-        -----
-        Based on ObservablesFromText from ImPlaneIA, e.g.
-        https://github.com/anand0xff/ImPlaneIA/blob/master/nrm_analysis/misctools/implane2oifits.py#L32
         """
         self.fringe_fitter = fringefitter
         self.n_holes = 7
 
         self.nslices = len(self.fringe_fitter.nrm_list)  # n ints
         self.n_baselines = int(comb(self.n_holes, 2))  # 21
         self.n_closure_phases = int(comb(self.n_holes, 3))  # 35
-        self.n_closure_amplitudes = int(comb(self.n_holes, 4)) # also 35
+        self.n_closure_amplitudes = int(comb(self.n_holes, 4))  # also 35
 
         self.method = method
         if self.method not in ["mean", "median", "multi"]:
-            msg = ('method for saving OIFITS file must be one of ["mean","median","multi"]. '
-                   'Defaulting to use mean!')
+            msg = (
+                'method for saving OIFITS file must be one of ["mean","median","multi"]. '
```

Comment:
This is a curious case - I'm not sure what is easier to parse or if the auto-formatter is choosing the correct fix here.

Reply:
I agree it's probably not any more readable, and perhaps less readable.  I think it saw that the first line was longer than 88 lines, but it wasn't smart enough to move part of the string into the second line.  So instead it broke it up over more lines.

I will change these specific lines.  I'm not sure there's a way to catch cases like this, short of turning off large portions of the line length auto-fixes.  Are you suggesting that I try that?  (happy to do so and see what happens, just wondering if your comment is a suggested fix to these lines only, or a suggested change to the auto-formatter)